src/auth.js

```diff
@@ -0,0 +1,135 @@
+const validationMessage = {
+  requiredEmail: "이메일을 입력해주세요",
+  invalidEmail: "잘못된 이메일 형식입니다.",
+  requiredPassword: "비밀번호를 입력해주세요.",
+  weekPassword: "비밀번호를 8자 이상 입력해주세요.",
+  mismatchedPassword: "비밀번호가 일치하지 않습니다.",
+  requiredNickname: "닉네임을 입력해주세요.",
+};
+
+const isValidEmail = (email) => {
```

Comment:
P2:
함수라는 것을 알 수 있도록 동사로 시작하는 함수명을 추천드립니다.
이름은 참고만 해주세요~

```suggestion
const checkEmailValid = (email) => {
```